Description: Remove deprecated use of np.float
Author: Marcel Stimberg <marcel.stimberg@inserm.fr>
Bug-Debian: https://bugs.debian.org/1027193
Applied-Upstream: 61ef84b316a3d0a892298adf51abd8ac50900758
Last-Update: 2023-01-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- brian.orig/brian2/units/fundamentalunits.py
+++ brian/brian2/units/fundamentalunits.py
@@ -1597,7 +1597,7 @@
         unitless = np.array(self / best_unit, copy=False)
         threshold = np.get_printoptions()['threshold'] // 100
         if unitless.ndim == 0:
-            sympy_quantity = np.float(unitless)
+            sympy_quantity = float(unitless)
         elif unitless.ndim == 1:
             array_str = np.array2string(unitless, separator=" & ", threshold=threshold,
                                         max_line_width=sys.maxsize)
